provider/spark_test.go

```diff
@@ -2015,28 +1888,28 @@ func sparkTestMaterializationUpdate(t *testing.T, store *SparkOfflineStore) {
 			// Have to expect int64(0) as it is the default value
 			// if a resource does not have a set timestamp
 			ExpectedSegment: []any{
-				TestRecordInt{Entity: "a", Value: 1, TS: int64(0)},
```

Comment:
Why are we removing the timestamp?

Reply:
There are two "default" 0 timestamps, January 1st year 0 and January first 1970. When we write year 0 to a file, it's the latter, but when the SQL statement creates a "dummy" 0 timestamp for materializations with no set timestamp, it's the former. These changes were made to make sure the outcome matched up